bentoml/cli/__init__.py

```diff
@@ -30,15 +34,22 @@ def cli():
 @cli.command()
 @click.option('--model-path', type=click.Path(exists=True), required=True)
 @click.option('--port', type=click.INT)
-def serve(model_path, port):
+@click.option('--storage-type', type=click.STRING)
```

Comment:
why is this option necessary? you can tell it by looking at model-path right?

Reply:
It is easy to detect. Still want user explicitly declare it, so they are aware of it.  Also, when we support more remote storage type, this will come handy

Reply:
agree but better to make it behave similar to other data science toolkits, e.g.
http://pandas.pydata.org/pandas-docs/stable/user_guide/io.html#reading-remote-files